src/__test__/components/plots/styling/SingleGeneSelection.test.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import _ from 'lodash';
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+
+import fakeConfig from '__test__/data/componentConfigs/embeddingContinuous.json';
+
+import SingleGeneSelection from 'components/plots/styling/SingleGeneSelection';
+
+const loadAndRenderSingleGeneSelection = async (config, setSearchedGene) => {
+  await act(async () => {
+    render(
+      <SingleGeneSelection
+        config={config}
+        setSearchedGene={setSearchedGene}
+      />,
+    );
+  });
+};
+
+describe('SingleGeneSelection', () => {
+  it('Shows gene received in props initially', async () => {
+    await act(async () => {
```

Comment:
Why do you wrap this in act too? The rest of the calls are not

Reply:
oh, I shouldn't wrap those in `act`, inside them there is an `act`